src/target/adiv5.c

```diff
@@ -743,8 +772,7 @@ void adiv5_dp_init(ADIv5_DP_t *dp)
 	dp->mem_write_sized = firmware_mem_write_sized;
 #endif
 
-	volatile uint32_t ctrlstat = 0;
-	volatile struct exception e;
+	volatile uint32_t ctrlstat;
```

Comment:
Two points - why is this volatile, and why not = 0?

Reply:
actually don't know why I touched that and it should remain initialized to 0, as for volatile, I don't know

Reply:
We can't see any reason it should be marked volatile

Reply:
heres the reason
`error: variable ‘ctrlstat’ might be clobbered by ‘longjmp’ or ‘vfork’ [-Werror=clobbered]`

Reply:
Ok, that's a good explanation.. it indicates an underlying bigger problem, but we'll take it for now. Please add a note about this to the definition.